models/Drink.py

```diff
@@ -0,0 +1,50 @@
+from services.Embedder import DrinkEmbedder
+from services.Sanitizer import DrinkSanitizer
+
+'''
+Drink Object
+    Processes json data retrieved from the request object and
+    stores a Discord.Embed object
+
+    TODO: Consider rewriting this as a functional module vs a Class
+'''
+class Drink():
+
+    def __init__(self, drink_json: dict, embedder: DrinkEmbedder, sanitizer: DrinkSanitizer):
+        _json = sanitizer.clean_filth(drink_json)
+        self._img = _json.get('strDrinkThumb')
+        self._name = _json.get('strDrink')
+        self._category = _json.get('strCategory')
+        self._alcoholic = _json.get('strAlcoholic')
+        self._glass = _json.get('strGlass')
+        self._ingredients_string = self.__get_ingredient_string(_json)
+        self._instructions = _json.get('strInstructions')
+        self.embed = embedder.embed(self)
+
+    def __get_ingredient_string(self, json) -> list:
```

Comment:
Did you intend to use `__get_ingredient_string` here instead of `_get_ingredient_string` for the name mangling? Not saying there's a problem with it or it should be changed, just want to understand if there's a particular reason for the double underscores.

Reply:
Really not much of reason - thought process was I saw no reason to keep it as a protected class if the model wasn't going to have children classes inherited from it. Making it private just seemed like it was a better fit